phpMain/arithmatic.php

```diff
@@ -45,3 +27,21 @@ function calculus ($a, $b) {
 }
 
 echo calculus (9, 3);
+
+// Attempt at complete function? WIP
+
+function calculus ($a, $b) {
+$sum= $a + $b;
```

Comment:
It is good to use indentation here. The code is easier to read that way. For e.g
```
function a() {
    $x = $y / 12;
    $blah = $x * 12;
}
```